Source/WebCore/rendering/RenderLayerCompositor.cpp

```diff
@@ -752,8 +754,8 @@ void RenderLayerCompositor::customPositionForVisibleRectComputation(const Graphi
 
     FloatPoint scrollPosition = -position;
 
-    if (m_renderView.frameView().scrollBehaviorForFixedElements() == ScrollBehaviorForFixedElements::StickToDocumentBounds)
-        scrollPosition = m_renderView.frameView().constrainScrollPositionForOverhang(roundedIntPoint(scrollPosition));
+    if (m_renderView.protectedFrameView()->scrollBehaviorForFixedElements() == ScrollBehaviorForFixedElements::StickToDocumentBounds)
```

Comment:
Can we store m_renderView.frameView() in a local variable instead of calling m_renderView.protectedFrameView twice?